### ISSUE TITLE
Request for enhanced documentation for svelte

### ISSUE THREAD

**Issue by maku** [nxext/nx-extensions#141]
I would really like to use this extension with a svelte app. 

But I do not  get it how to handle a "custom" rollup config. I read in the docu that there is "--sveltePreprocessConfig" and "--rollupConfig" option for the build step. But I have not found anything how to use this. 

Is there documentation or an example project?

**Comment by DominikPieper** [comment]
Yes, unfortunately there's not that much documentation yet. There's a issue with a example config, the options have to be a path to a js file which manipulates the config.

**Comment by DominikPieper** [comment]
Here it is https://github.com/nxext/nx-extensions/issues/86

**Comment by DominikPieper** [comment]
That's a example in there for the preprocessor config. Just paste it in a js file and add it to the project and add the option with the path

**Comment by akshay-nm** [comment]
So you create a `rollup.config.js` in project dir, and specify the path to it in `svelte.config.cjs`?
